web/src/people.ts

```diff
@@ -118,6 +118,45 @@ export function init(): void {
 // WE INITIALIZE DATA STRUCTURES HERE!
 init();
 
+// A function that sorts Email according to the user's full name
+export function sort_emails(emails: string[], user_ids: number[]): string[] {
+    const namesArray = user_ids.map((user_id) => {
+        let person;
+        if (!people_by_user_id_dict.has(user_id)) {
+            person = undefined;
+        } else {
+            person = people_by_user_id_dict.get(user_id);
+        }
+        if (!person) {
+            return "?";
+        }
+        return person.full_name;
+    });
+    // Combine the non-undefined elements
+    const combinedArray = namesArray
+        .map((name, index) => ({
+            name,
+            email: emails[index],
+        }))
+        .filter((item) => item.name !== undefined);
+    // Sort the combined array based on the 'name' property.
+    combinedArray.sort((a, b) => {
+        const nameA = a.name.toLowerCase(); // Use ! to assert that a.name is not undefined
+        const nameB = b.name.toLowerCase(); // Use ! to assert that b.name is not undefined
+        return nameA.localeCompare(nameB);
+    });
+    const sortedemails = combinedArray.map((item) => item.email);
+    return sortedemails;
```

Comment:
I am not sure whether this function is needed or not? A couple of places where you call `sort_emails` already use `get_pm_full_names` to get the names eventually and the names are sorted there.

I would suggest to check whether we actually require emails sorted by names there and if there are only one or two such instances, then we can just handle them separately without this function.

Reply:
And there are variables that still use camelCase, so if you come to a conclusion that we need this function please fix them as well.

Reply:
There are 3 functions where I am calling ```sort_emails``` function. In that functions, emails are being received by ids of users not by their sorted names. To sort them by their names, first we have to get their names by id and then sort them accordingly. If we treat all 3 instances separately, it will increase unnecessary repetitiveness of the code. That's why I feel that function ```sort_emails``` is useful and we should keep this.

I modifies the camelCase variables to snake_case.